scripts/python/database.py

```diff
@@ -58,19 +58,18 @@
     image_id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
     name TEXT NOT NULL UNIQUE,
     camera_id INTEGER NOT NULL,
-    prior_qw REAL,
```

Comment:
What happens if we open an old database? Does it crash?

Reply:
OK, I guess the migration script below answers the question. It will crash but can be migrated with the script?

Reply:
It does not crash, it just ignores the columns. Which is why the `--cleanup` option in the migration script is optional.

Reply:
Thanks